IQSS/dataverse
# make BagIT source organization info (name, address, email) configurable

## **What this PR does / why we need it**:

Currently values are hard-coded in Bundle.properties.

## **Which issue(s) this PR closes**:

- Closes #8760

## **Special notes for your reviewer**:

I tested this with MPCONFIG in Docker. I would add an API test for it but I don't think it's possible. I opened this issue about that:

- #10128

The following note is old. I left it in but the bottom line is that I suggesting `,` instead of `\n` per the RFC.

While testing with these values...

```
DATAVERSE_BAGIT_SOURCEORG_NAME=LibraScholar
DATAVERSE_BAGIT_SOURCEORG_ADDRESS=123 Wisdom Way\nCambridge, MA\nUSA
DATAVERSE_BAGIT_SOURCEORG_EMAIL=hello@dataverse.librascholar.edu
```

... I get this output...

```
Source-Organization: LibraScholar
Organization-Address: 123 Wisdom Way\nCambridge, MA\nUSA
Organization-Email: hello@dataverse.librascholar.edu
```

Do we want the address on all one line? Maybe with commas?

Update: the example at https://datatracker.ietf.org/doc/html/rfc8493 uses commas (`1 Main St., Cupertino, California, 11111`) so I'm going with that and mentioned it in our guides.

## **Suggestions on how to test this**:

- use the new config (either as JVM options with MPCONFIG)
- export a bag
- unzip it
- confirm values are correct in bag-info.txt

## **Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

No.

## **Is there a release notes update needed for this change?**:

Yes, included.

## **Additional documentation**:

Included: https://dataverse-guide--10122.org.readthedocs.build/en/10122/installation/config.html#bag-info-txt
